test/provision/k8s_install.sh

```diff
@@ -41,6 +41,10 @@ source ${PROVISIONSRC}/helpers.bash
 sudo bash -c "echo MaxSessions 200 >> /etc/ssh/sshd_config"
 sudo systemctl restart ssh
 
+retry_function "wget https://get.helm.sh/helm-v2.14.2-linux-amd64.tar.gz"
```

Comment:
Would be good for us to containerize `helm` for the VMs so developers can run this locally as well. This can be done in a follow-up PR.

Reply:
Yes. That was idea. Validate this and then include it into the box.